pkg/vm/engine/disttae/txn.go

```diff
@@ -362,12 +362,12 @@ func blockRows(meta BlockMeta) int64 {
 	return meta.Rows
 }
 
-func blockMarshal(meta BlockMeta) []byte {
+func BlockMarshal(meta BlockMeta) []byte {
```

Comment:
can we get rid of marshal/unmarshal?

Reply:
will be fixed in next pr

Reply:
I think this marshal is a vector-like serialization, this thing should be lossless after the degob is completed, there is no need to remove. @badboynt1  @fengttt I wonder when degob will be finished. In fact if that pr is longer, you can change the marshal and unmarshal here to pointer operations first.